src/main/java/io/github/thebusybiscuit/exoticgarden/ExoticGarden.java

```diff
@@ -69,6 +69,7 @@
 	private Category magicalCategory;
 	private Kitchen kitchen;
 
+	@SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
Not really a fan of these SuppressWarnings things, Id rather have it removed and let the warning stay unless it is fixed.